debian/rules

```diff
@@ -1,103 +1,21 @@
 #!/usr/bin/make -f
 
-dist_release := $(shell lsb_release -cs)
-
-dh_extra_flags = -plandscape-common -plandscape-client
-
-include /usr/share/python3/python.mk
-ifeq (,$(py_sitename))
-	py_sitename = site-packages
-	py_libdir = /usr/lib/python$(subst python,,$(1))/site-packages
-	py_sitename_sh = $(py_sitename)
-	py_libdir_sh = $(py_libdir)
-endif
+export DH_ALWAYS_EXCLUDE=.git
+export DH_VERBOSE = 1
 
 package = landscape-client
 root_dir = debian/tmp
 SHAREDIR = /usr/share/landscape
 LIBDIR = /usr/lib/landscape
 
-revision = $(shell dpkg-parsechangelog | grep ^Version | cut -f 2 -d " "| cut -f 2 -d "-")
-
-landscape_common_substvars = debian/landscape-common.substvars
-landscape_client_substvars = debian/landscape-client.substvars
-
-build-arch: build
+%:
+	dh $@ --with python3 --buildsystem=pybuild
 
-build-indep: build
-
-build: build-stamp
-build-stamp:
-	dh_testdir
-	sed -i -e "s/^DEBIAN_REVISION = \"\"/DEBIAN_REVISION = \"-$(revision)\"/g" landscape/__init__.py
```

Comment:
That version is used as user-agent version to identify regressions from the server side. I think we should keep it. Not sure how's the proper way do to it, but if I had to, 
```
override_dh_auto_build:
       sed -i -e "s/^DEBIAN_REVISION = \"\"/DEBIAN_REVISION = \"-$(shell dpkg-parsechangelog | sed -rne 's,^Version: ([^-]*)-(.*),\2, p')\"/g" landscape/__init__.py
       dh_auto_build
```